src/Entities/RulesEngine/Actions/SendEmail/EmailNotification.php

```diff
@@ -45,16 +45,6 @@ public function getVersion(): string
         return $this->getAttribute('version');
     }
 
-    /**
-     * @param string $value
-     *
-     * @return $this
-     */
-    public function setVersion($value): self
```

Comment:
But this exists in API still?

Reply:
It is, but it is readonly

Reply:
Not sure this is it, but here it's not readonly https://github.com/Rebilly/RebillyUserAPI/blob/master/spec/components/schemas/Rules/RulesEmailNotification.yaml#L7